test/e2e/components/vl-alert.js

```diff
@@ -12,7 +12,22 @@ class VlAlert extends VlElement {
   }
 
   async getTitle() {
-    return this._innerText('#title');
+    const element = (await this._getTitleSlotElements())[0];
+    if (element) {
```

Comment:
hoe komt het eigenlijk dat we hier deze if case introduceren

Reply:
Afhankelijk van slot/attribuut moet je de content anders gaan benaderen.